CMakeLists.txt

```diff
@@ -602,19 +602,27 @@ if (BUILD_TESTING)
         target_link_libraries(fuzztest PUBLIC ${PROJECT_NAME})
 
         # Set default values for fuzzing if not defined
-        if(NOT DEFINED FUZZ_TIMEOUT_SEC)
+        if(DEFINED ENV{FUZZ_TIMEOUT_SEC})
+            set(FUZZ_TIMEOUT_SEC $ENV{FUZZ_TIMEOUT_SEC})
+        else()
             set(FUZZ_TIMEOUT_SEC 60)
         endif()
 
-        if(NOT DEFINED CORPUS_UPLOAD_LOC)
+        if(DEFINED ENV{CORPUS_UPLOAD_LOC})
+            set(CORPUS_UPLOAD_LOC $ENV{CORPUS_UPLOAD_LOC})
+        else()
             set(CORPUS_UPLOAD_LOC "none")
         endif()
 
-        if(NOT DEFINED ARTIFACT_UPLOAD_LOC)
+        if(DEFINED ENV{ARTIFACT_UPLOAD_LOC})
+            set(ARTIFACT_UPLOAD_LOC $ENV{ARTIFACT_UPLOAD_LOC})
+        else()
             set(ARTIFACT_UPLOAD_LOC "none")
         endif()
 
-        if(NOT DEFINED FUZZ_TESTS)
+        if(DEFINED ENV{FUZZ_TESTS})
```

Comment:
Have you tested that this is how you're actually supposed to be setting these variables with CMAKE?

Reply:
Yes, otherwise these values wouldn't be read from the environment, and this is the same way we handle environment variables elsewhere in our CMake.